server/broadcast.go

```diff
@@ -81,23 +100,29 @@ func (pi *packetInfo) announceStart() {
 	// If the user has disconnected, then no need to send
 	// the broadcast.
 	if sender == nil {
+		log.Debugf("[ANNOUNCE] Ignoring message from %s because player no longer exists\n", getIP(pi.conn))
```

Comment:
Should add prefix for this to be consistent. =)

Reply:
Done above f8a49a2